datadog/resource_datadog_synthetics_test_.go

```diff
@@ -444,6 +419,35 @@ func syntheticsTestOptionsList() *schema.Schema {
 	}
 }
 
+func syntheticsConfigVariable() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		Optional: true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"example": {
+					Type:     schema.TypeString,
```

Comment:
Can you add a description to each of these fields (in each schema)? We have an ongoing effort to ensure all fields have descriptions so we can work on autogenerating the terraform documentation - #798